Error: Cannot find module '../dist/index.js'

Hmmm... Tried npm updating on a Pi, then removing and reinstalling. After install, when running I get the same error both ways. That is: Error: Cannot find module '../dist/index.js'

Looking around /usr/lib/node_modules/tallyarbiter, there is no dist directory

It's possible there is cruft from a previous install, but I'd like to avoid rebuilding the Pi.

_Originally posted by @wrldtvlr in https://github.com/josephdadams/TallyArbiter/discussions/277#discussioncomment-1716970_

--- a/package.json
+++ b/package.json
@@ -38,6 +38,7 @@
         "package-lock.json",
         "index.js",
         "bin",
+        "dist",
         "ui-dist",
         "readme.md",
         "LICENSE"

